app/src/app/services/content-service-extension.service.ts

```diff
@@ -15,7 +15,17 @@ import { take } from 'rxjs/operators';
 })
 export class ContentServiceExtensionService {
   constructor(private appConfigService: AppConfigService) {
-    this.updateContentServiceAvailability();
+    this.prepareContentServiceProperty();
+  }
+
+  private prepareContentServiceProperty() {
+    if (!this.existsContentServicePropertyLocalStorage()) {
```

Comment:
hmm, I think that this call prevents checking the app config service in case the flag was previously stored in the local storage. that could mean that if admin changes this setting on the backend, the app will still be using localstorage....

Reply:
This is strange, because there are other options not have this behavior (don't lose the localstorage property after page refresh):
- Enable AI Extensions
- Enable Process Services Extensions

The only ones with this "bug" - lose/reset this localstorage property after page refresh:
- Enable Content Services Extensions
- Enable Microsoft Office Online Extension (I noticed that it is like that too)